PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+## Description
+
+Please describe your pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue. When linking to an issue, please use `refs #...` in the description of the pull request.
```

Comment:
Not sure I fully understand the point of specifically using `refs #...`? What's wrong with Github's `#...` notation (or just a link, which I think Github will still figure out)?

Reply:
oh yes, Github's `#...` should work :)